packages/admin-ui/ui/public/locales/ar/translation.json

```
@@ -0,0 +1,1919 @@
+{
+  "back-button-go-back": "رجوع",
+  "sales-channels-display-available-count": "متاح في <2>{{availableChannelsCount}}</2> من <6>{{totalChannelsCount}}</6> قنوات المبيعات",
+  "activity-drawer-activity": "النشاط",
+  "activity-drawer-no-notifications-title": "هادئ هنا...",
+  "activity-drawer-no-notifications-description": "ليس لديك أي إشعارات حاليًا ، ولكن بمجرد أن يكون لديك إشعارات ستظهر هنا.",
+  "activity-drawer-error-title": "آه لا...",
+  "activity-drawer-error-description": "حدث خطأ ما أثناء محاولة جلب إشعاراتك - سنواصل المحاولة!",
+  "activity-drawer-processing": "جار المعالجة...",
+  "analytics-config-form-title": "جعل بيانات الاستخدام الخاصة بي مجهولة الهوية",
+  "analytics-config-form-description": "يمكنك اختيار جعل بيانات استخدامك مجهولة الهوية. إذا تم تحديد هذا الخيار ، فلن نقوم بجمع معلوماتك الشخصية ، مثل اسمك وعنوان بريدك الإلكتروني.",
+  "analytics-config-form-opt-out": "عدم مشاركة بيانات الاستخدام الخاصة بي",
+  "analytics-config-form-opt-out-later": "يمكنك دائمًا اختيار عدم مشاركة بيانات الاستخدام الخاصة بك في أي وقت.",
+  "analytics-preferences-success": "نجاح",
+  "analytics-preferences-your-preferences-were-successfully-updated": "تم تحديث تفضيلاتك بنجاح",
+  "analytics-preferences-error": "خطأ",
```

```suggestion
  "analytics-preferences-error": "حدث خطأ",
```

I think it makes sense to use this translation for all "Error" translations